### ISSUE TITLE
Handle decryption errors more gracefully

### ISSUE THREAD

**Issue by woodruffw** [woodruffw/kbs2#5]
Decryption error messages are currently leaked to the user with the `rage` backend (and probably `age` as well):

```bash
$ echo junk > ~/.local/share/kbs2/junk
$ kbs2 pass junk
Error: failed to fill whole buffer

[ Did rage not do what you expected? Could an error be more useful? ]
[ Tell us: https://str4d.xyz/rage/report                            ]
Fatal: EOF while parsing a value at line 1 column 0
```

These should be hidden and propagated more cleanly.
